pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -101,6 +105,72 @@ def syncD2H(self, use_async=False):
         self._kokkos_state.DeviceToHost(self._state.ravel(order="C"))
         self._pre_rotated_state = self._state
 
+    """
```

Comment:
Is this meant to be commented out?

Reply:
yea, i left both cpp and python for your opinion.